CRM/Member/Form/Membership.php

```diff
@@ -1700,7 +1686,7 @@ public function submit() {
       $formValues['receipt_text_signup'] = $formValues['receipt_text'];
       // send email receipt
       $this->assignBillingName();
-      $mailSend = self::emailReceipt($this, $formValues, $membership);
+      $mailSend = $this->emailMembershipReceipt($formValues, $membership);
```

Comment:
I don't think $mailSend is ever used?  Also, emailMembershipReceipt is returning an array and $mailSend is the second element...

Reply:
ok